.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: rust
+rust: stable
+cache: cargo
+
+before_script:
+    - export PATH="$PATH:$HOME/.cargo/bin"
+
+script:
+    # By installing rustfmt, auto-generated code becomes more readable.
```

Comment:
We install `rustfmt` but don't seem to use it?

Reply:
Edd confirms that without the use of rustfmt, auto-generated code contains lines so long that they cause Travis problems.

Reply:
A possible comment might be something like "We need to use rustfmt on auto-generated code: without it, code is put on a single line, which then causes Travis to crash because ...".